SteamList/Controllers/GameDetailsViewController.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  GameDetailsViewController.swift
+//  SteamList
+//
+//  Created by Macbook on 03/01/2023.
+//
+
+import UIKit
+
+class GameDetailsViewController: UIViewController {
+ 
+    private let game: Game
+    
+    @IBOutlet weak var favIcon: UIImageView!
+    @IBOutlet weak var gameName: UILabel!
+    
+    fileprivate func configOutlets() {
```

Comment:
I would re-organize method positions a little bit to make it easy to read and understand. Usually, you can follow the order below:

* initializer
* view controller lifecycle methods
* private methods
* etc...

Reply:
done